doc/user/logging.md

```diff
@@ -60,6 +60,54 @@ $ go run main.go --zap-level=1
 {"level":"info","ts":1559866310.065119,"logger":"scoped","msg":"Printing at INFO level"}
 {"level":"debug","ts":1559866310.065123,"logger":"scoped","msg":"Printing at DEBUG level"}
 ```
+## Custom zap logger
+
+In order to use a custom zap logger, [`zapr`][repo_zapr] can be utilized to wrap it in a logr implementation.
+
+Below is an example illustrating the use of [`zap-logfmt`][logfmt_repo] in logging. Here, the `NewLogger` function produces a new `logr.Logger` from an existing `zap.Logger`.
+
+### Example
+
+```Go
+package main
+
+import (
+	"github.com/go-logr/zapr"
+	zaplogfmt "github.com/sykesm/zap-logfmt"
+	"go.uber.org/zap/zapcore"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log logr.Logger
+
+func main() {
+	configLog := zap.NewProductionEncoderConfig()
+	configLog.EncodeTime = func(ts time.Time, encoder zapcore.PrimitiveArrayEncoder) {
+		encoder.AppendString(ts.UTC().Format(time.RFC3339))
+	}
+
+	logger := zap.New(zapcore.NewCore(
+		zaplogfmt.NewEncoder(configLog),
+		os.Stdout,
+		zapcore.DebugLevel,
+	))
+
+	// Construct a new logr.logger. 
+	log = zapr.NewLogger(logger)
+
+	// Set the controller logger to log, which will
+	// be propagated through the whole operator, generating
+	// uniform and structured logs.
+	logf.SetLogger(log)
+}
+```
+
+#### Output using custom zap logger
+```console
+$ go run main.go
+ts=2020-02-25T22:42:43Z level=info msg="Printing at INFO level"
+ts=2020-02-25T22:42:43Z level=info msg="Printing at INFO level"
```

Comment:
The logs shows INFO level, but the example is adding `zapcore.DebugLevel`. So, is this logs from the above example? Should not it bee debug level?